src/libraries/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -138,5 +139,6 @@ System.Net.Http.WinHttpHandler</PackageDescription>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Runtime.InteropServices" />
```

Comment:
Why do we need this? How are there DllImports if it didn't have access to `System.Runtime.InteropServices`?

Reply:
Same question for System.IO.Ports.csproj.